apis/placement/v1beta1/stageupdate_types.go

```diff
@@ -0,0 +1,469 @@
+/*
+Copyright (c) Microsoft Corporation.
+Licensed under the MIT license.
+*/
+
+package v1beta1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// +genclient
+// +genclient:Cluster
+// +kubebuilder:object:root=true
+// +kubebuilder:subresource:status
+// +kubebuilder:resource:scope=Cluster,categories={fleet,fleet-placement},shortName=crsur
```

Comment:
do you want to build some printcolumn to show the status?

Reply:
Let me think about it and update in a separate PR.